app/services/external_upload_manager.rb

```diff
@@ -33,7 +33,7 @@ def self.create_direct_upload(current_user:, file_name:, file_size:, upload_type
 
     upload_stub =
       ExternalUploadStub.create!(
-        key: key,
+        key: key.gsub("#{SiteSetting.s3_upload_bucket}/", ""),
```

Comment:
TODO: Need to figure out a better way of doing this; the key is doubling up the bucket in the path and we don't want to store it in the DB that way.